src/Sql/Sql/DistributedAvailabilityGroup/Cmdlet/NewAzureSqlManagedInstanceLink.cs

```diff
@@ -123,16 +150,8 @@ public override void ExecuteCmdlet()
                 default:
                     break;
             }
+            base.ExecuteCmdlet();
 
-            // messages describing behavior with -WhatIf and -Confirm flags
```

Comment:
We should support should process for New-Az* cmdlet. Please revert these changes.

Reply:
we're still supporting it - via [AzureSqlCmdletBase](https://github.com/Azure/azure-powershell/pull/19059/files#diff-8f33295ee2f2df46a431874d5cd802002e0d98732b0087738f0da851045b0209R149)

Reply:
The problem is not all commands inherited from AzureSqlCmdletBase should support shouldprocess, like Get-Az* cmdlets, see https://cs.github.com/AuxMon/azure-powershell/blob/17a2a45099e51f9420f4705f820667b0378fd610/src/Sql/Sql/DataClassification/Cmdlet/GetAzSqlInstanceDatabaseSensitivityRecommendation.cs?q=%3A+AzureSqlCmdletBase%3C#L28

Reply:
they don't support it magically; you still need to explicitly specify `SupportsShouldProcess = true`